README.md

```diff
@@ -1,119 +1,23 @@
-Idris 2
-=======
+# Idris 2
 
 [![Documentation Status](https://readthedocs.org/projects/idris2/badge/?version=latest)](https://idris2.readthedocs.io/en/latest/?badge=latest)
 [![Build Status](https://github.com/idris-lang/Idris2/actions/workflows/ci-idris2.yml/badge.svg)](https://github.com/idris-lang/Idris2/actions/workflows/ci-idris2.yml)
 
-[Idris 2](https://idris-lang.org/) is a purely functional programming language
-with first class types.
+[Idris 2](https://idris-lang.org/) is a purely functional programming language with first class types.
 
-For full installation instructions, see [INSTALL.md](INSTALL.md).  Briefly, if
-you have Chez Scheme installed, with the executable name `chez`, type:
```

Comment:
I removed the default "install from sources" stuff because I don't think most people will install from sources. Is that accurate?

Reply:
I don't think having a tl;dr for installing from sources is bad, due to the stuff I mentioned in my other comment.